nixos/modules/services/networking/dnscrypt-proxy.nix

```diff
@@ -277,7 +277,8 @@ in
         ExecStart = "${dnscrypt-proxy}/bin/dnscrypt-proxy ${toString daemonArgs}";
         Restart = "always";
         User = "dnscrypt-proxy";
-
+        CapabilityBoundingSet = "CAP_NET_BIND_SERVICE";
```

Comment:
It absolutely is redundant :)

From what I understand, AC adds CAP_NET_BIND to the capabilities of a process (by inheriting them), and CBS restricts the possible capabilities to be solely CAP_NET_BIND; it is merely an extra safeguard in case systemd screws up.